src/runtime.rs

```diff
@@ -185,21 +658,21 @@ impl Worker {
         }
     }
 
-    fn probe(&mut self, rt: &runtime::RuntimeMetrics, sample: &mut Sample) {
+    fn probe(&mut self, rt: &runtime::RuntimeMetrics, Metrics: &mut Metrics) {
```

Comment:
I don't think capital `M` (in `Metrics`) here is idiomatic. It should be `metrics: &mut Metrics`.

Reply:
Whoops, that's a rename gone awry.